src/apps/material/helper.ts

```diff
@@ -402,3 +410,75 @@ export const isParallelReservation = (manifestations: Manifestation[]) =>
   manifestations.length > 1 &&
   hasCorrectAccessType(AccessTypeCode.Physical, manifestations) &&
   !isArticle(manifestations);
+
+// Because we  need to exclude the branches that are blacklisted, we need to use a custom hook to prevent duplicate code
+const getBlacklistedArgs = (
+  faustIds: FaustId[],
+  config: UseConfigFunction,
+  blacklist:
+    | "blacklistedAvailabilityBranchesConfig"
+    | "blacklistedPickupBranchesConfig"
```

Comment:
I dont think we need to use the full config prop names here. I think it could be more need to just make a union of some simple keys like: `"availability" | "pickupBranches" `.
Then you can construct the config names like they should be inside this function either with a map or a pattern. does it make sense?